lending/loan_management/doctype/loan_repayment/loan_repayment.py

```diff
@@ -671,6 +672,14 @@ def validate_amount(self, amounts):
 					)
 				)
 
+	def validate_open_disbursement(self):
+		loan_disbursement_status = frappe.get_value(
+			"Loan Disbursement", self.loan_disbursement, "status"
+		)
+		print(loan_disbursement_status)
```

Comment:
Remove print statement
Also run the validation/make db call if loan_disbursement link is present. Skip if loan disbursement is not entered